python/myutils/HiggsCandidateSystematics.py

```diff
@@ -51,13 +66,29 @@ def customInit(self, initVars):
 
         for p in ['Jet_pt_minmax', 'Jet_mass_minmax']:
             for q in ['Up', 'Down']:
-                self.addVectorBranch(p+q, length=self.nJetMax, leaflist=p+q+'[nJet]/F')
+                self.branchBuffers[p+q] = array.array('f', [0.0]*self.nJetMax)
+                self.branches.append({'name': p+q, 'formula': self.getVectorBranch, 'arguments': {'branch': p+q}, 'length': self.nJetMax, 'leaflist': p+q+'[nJet]/F'})
+                ## wrong values when using this. Use two lines above instead.
```

Comment:
@piberger I had to change this line otherwise Jet_pt_minmax and Jet_mass_minmax are broken.